R/translate.R

```diff
@@ -156,12 +180,14 @@ translate_part <- function(xml,
   woolish <- tinkr::yarn$new(path = temp_file)
   woolish$body <- fakify_xml(xml)
 
-  ## protect content inside curly braces ----
+  # protect inline math
+  woolish$body <- tinkr::protect_math(woolish$body)
```

Comment:
So, why couldn't this function save us the trouble of digesting the equations? 

1. use `tinkr::protect_math()`
2. make the "asis" text tags, tags of a given name "asis".
3. add the exclusion to the API call

So actually closer to what happens to curly braces: https://github.com/ropensci-review-tools/babeldown/blob/c528675613e2ae7f234a4efc60a9ec8e5a785bbe/R/translate.R#L159

https://github.com/ropensci-review-tools/babeldown/blob/c528675613e2ae7f234a4efc60a9ec8e5a785bbe/R/translate.R#L368

Reply:
ok, I am not familiar with the use of xml (and xml2 package), but I start to to see the logic.
I used `tinkr::protect_math` (step 1), but not step 2 an 3. I'll try to understand what's happening with curly

Reply:
I did it the way you suggest, simpler. However, I have a problem with $f$ expressions.